app.rb

```diff
@@ -199,7 +199,7 @@ def ipxe_boot_url
         net_id = "net#{index - 1}"
         vars[net_id] = "${#{net_id}/mac:hexhyp}"
       end
-      ["dhcp_mac", "serial", "asset", "uuid"].each { |k| vars[k] = "${#{k}}" }
+      ["dhcp_mac", "serial", "asset", "uuid", "ip_addr"].each { |k| vars[k] = "${#{k}}" }
```

Comment:
Is there a reason not to include this, regardless of the subnet functionality included in this PR? I'm not sure if this value is absent in any cases, or why it was omitted originally.